requirements/requirements.in

```diff
@@ -7,7 +7,7 @@ azure-keyvault
 boto3
 botocore
 channels
-channels-redis==3.4.1  # see UPGRADE BLOCKERs
+channels-redis
```

Comment:
Do we want to fully unpin this given its upgrade blocker status before?

Reply:
its fine